src/frontend/src/env/rest/blockchain.env.ts

```diff
@@ -0,0 +1,3 @@
+import { UrlSchema } from '$lib/validation/url.validation';
+
+export const BLOCKCHAIN_API_URL = UrlSchema.parse('https://blockchain.info');
```

Comment:
same as https://github.com/dfinity/oisy-wallet/pull/3264#discussion_r1825452517

Reply:
after that PR, i changed here too